ansible-ipi-install/roles/shared-labs-prep/tasks/main.yml

```diff
@@ -258,6 +258,14 @@
     mode: 0640
   become: yes
 
+- name: Configure dnsmasq service to start after network is online
```

Comment:
We probably need a daemon-reload after this, but since we are only handling this case on reboot, the daemon is reloaded anyway.